app/user_service/models.py

```diff
@@ -105,3 +105,36 @@ def alter_user(self, user):
         except Exception as e:
             self.dbconnect.rollback()
             raise e
+
+    def delete_user(self, data_loader, username):
+        '''remove user and all of its datasets'''
+
+        # remove user deletes every row that depends on it because of cascade deletion
+
+        cursor = self.dbconnect.get_cursor()
+
+        try:
+            # first drop all schemas owned by the user
+            query = cursor.mogrify('SELECT id FROM dataset WHERE owner = %s',
```

Comment:
When running this locally, I get the following error:

`column "owner" does not exist`
`LINE 1: SELECT id FROM dataset WHERE owner = 'sergio'`